docs/actions.rst

```diff
@@ -360,31 +359,56 @@ When a rule's condition(s) are satisfied we get the results back as:
 
 debug
 *****
-  Write the event to stdout
-  It accepts an arbitrary dict that it's injected into the event to be printed
-
-none
-****
-  No action, useful when writing tests
-  No arguments needed
-
-echo
-****
-.. list-table:: Write a user specified message to stdout
+.. list-table:: debug ansible-rulebook
    :widths: 25 150 10
    :header-rows: 1
 
    * - Name
      - Description
      - Required
-   * - message
-     - A terse message to display to stdout
-     - Yes
+   * - msg
+     - A simple string or an array of strings, which can have references to event or events
+     - No
+   * - var
+     - The variable to print, which can have references to event or events
+     - No
+
+| The debug action tries to mimic the debug command in ansible.
+| If no arguments are provided it prints the matching events along with other important properties
+| **msg** and **var** are mutually exclusive, you can have only 1 of them in the debug
+| msg can be a single string or an array of strings, with references to event or events
 
 Example:
 
-.. code-block:: yaml
+    .. code-block:: yaml
 
-    action:
-      echo:
-        message: Hello World
+       name: debug with single message
+       condition: event.i >= 5
+       action:
+          debug:
+            msg: Simple debug message
+
+
+    .. code-block:: yaml
+
+       name: debug with multiple messages
+       condition: event.i >= 5
+       action:
+          debug:
+            msg: 
+               - "Message 1 {{ event }}"
+               - Second Message
+
+    .. code-block:: yaml
+
+       name: debug with var
+       condition: event.i >= 5
+       action:
+          debug:
+            var: "{{ event.i }}"
```

Comment:
From this example and line 399 I don't see the difference var vs msg. Why not just use msg?

Reply:
@bzwei var doesn't need the curly braces, we should be able to just use event.i. And use dpath to pick up the data.
In the ansible documentation it says that var should not be using curly braces.
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/debug_module.html#parameter-var
I am going to be replacing the braces and just running dpath on anything in that field. By the time it comes to the debug method it has already gone thru jinja substitution.

Reply:
@bzwei I have removed the {{ }} which is optional